internal/kustomize/kustomize.go

```diff
@@ -22,6 +23,7 @@ func buildSetImageCmd(dir, fqImageRef string) *exec.Cmd {
 		"image",
 		fqImageRef,
 	)
+	cmd.Env = append(cmd.Env, os.Environ()...)
```

Comment:
This is actually the default when `cmd.Env == nil` but I've added it here explicitly so that any further `env` customization will be additive.